### DISCUSSION TITLE
[Feature]: Sorting for albums

### DISCUSSION THREAD

**Discussion by bo0tzz** [immich-app/immich#1689] [Feature Request]
### Feature detail

It would be nice to have a sort option for the albums view. Google Photos currently provides three such options:
* Most recent photo - Album with the newest photo first
* Last modified - Most recently modified album first
* Album title - Alphabetical

It would also be nice to have similar sorting options inside of albums (and possibly on the main photos feed) - things like:
* Age (oldest/newest first)
* Most recently added first
* Type (photos or videos first)

### Platform

Web

**Comment by matthinc** [comment]
First album sorting options will be included in one of the next mobile releases.

**Comment by alextran1502** [comment]
I think this is done, correct?

**Reply by scottrbaxter** [reply]
Can you better explain what you mean by done? Running server `v1.92.1` with iOS `v1.92.0` - If i open any album, i see no sorting options on mobile or in the webapp. Photos themselves appear to be automatically sorted newest to oldest. For albums, this really doesn't make sense. All of my albums are meant to be shared sequentially, (my typical album is several hundred photos over a single day), and I simply cannot find a way to sort "oldest to newest" with Immich. I've looked through github issues and also some of the code base, it seems there was a time when sorting was "oldest to newest" to some degree, but was changed a while back, correct? Then there appeared to be a code refactor, and I haven't back tracked exactly where that setting went, or how it's been refactored.

Given all of the amazing features already provided in this photo solution, I have a tough time believing that sorting photos "oldest to newest" in albums does not exist, but I simply cannot find the setting to do so... or better yet, include a setting for the "default sort method", for media in all albums.

**Comment by Parsiuk** [comment]
Manual photos order would be nice to have as well.

**Reply by x42x64** [reply]
I second that. Do we need to create a separate feature request for that?

**Reply by ok-kenji** [reply]
At this time, this is the only thing keeping me from moving into Immich full time. Being able to build a 'story' from images captured out of order is a must-have feature for Albums.

**Reply by motrengaw** [reply]
My use case for IMMICH is a secure and private image share for friends and family--emphasis on sharing. Manual--ideally drag-and-drop--ordering is vital to this as both I and my F&F will want to manipulate the "story line" of Album events.  I've purchased, as the build so far is clean and impressive, and UI will be easily grasped by F&F.  All that is lacking is Album sort options for me, particularly drag-and-drop a la Google/Picasa.  Taking it on faith that this obvious but no doubt challenging-to-implement feature is a priority.  Thanks!!

**Reply by chris400** [reply]
dfntly!
"Custom" is one of my high priorities

@alextran1502 would you mind chiming in? Esp. as you put a comment in here yourself stating that this feature might be completed. Which I slightly disagree with :)

**Reply by haayman** [reply]
I've scanned a lot of slides, so the exif dates are incorrect. Therefore I'd like to be able to sort it manually

**Reply by bequeem** [reply]
I opened a feature request going a bit more in depth [here](https://github.com/immich-app/immich/discussions/26595) because this was bothering be extremely.

**Reply by chris400** [reply]
There is a dedicated feature request for manual re-ordering. You might want to place your vote there
https://github.com/immich-app/immich/discussions/2208

**Comment by waclaw66** [comment]
A bug on mobile. While adding photos to the album the order of albums is from oldest to the recent one (at the end). Shared photos dropdown above shows them correctly.

**Comment by malconsei** [comment]
When ordering albums by "Most recent photo", web app 1.75.1 seems to sort by `lastModifiedAssetTimestamp` which in turn may come from the db field `assets.fileModifiedAt` - or anyway, today I geotagged and uploaded some very old pictures and their albums are right on top.

I think the common expectation is to sort by _file creation_ date. Would it be possible to change the feature this way?

**Reply by malconsei** [reply]
@klejejs I believe you developed the feature, could you chip in?

**Reply by klejejs** [reply]
Yes, that is the case. For the web, I copied the options from GPhotos - "last modified" and "album title" come from album data, but "most recent photo" checks assets in the album from DB and sorts albums by asset date.

**Reply by klejejs** [reply]
There is actually a PR for mobile already, but there is no progress on it :(
https://github.com/immich-app/immich/pull/3804

**Reply by malconsei** [reply]
Sorry for disappearing, I had put my experiments with immich on hold. I still think that the "most recent photo" should refer to the date it was taken and not modified/postprocessed.
I temporarily mitigated the problem for my collection with an `update assets set "fileModifiedAt" = "fileCreatedAt";` in the db, though I expect some jobs to overwrite it at some point. (For other users: Please don't try unless you know what you're doing, and backup your db first :) )

**Comment by chk86** [comment]
@bo0tzz just closed #4735 as a duplicate of this feature request, so I want to clarify: is adding sorting options inside individual albums still planned? Otherwise, I will create a separate request so this doesn't get lost.

**Reply by bo0tzz** [reply]
Yes, sorting inside albums is also covered by this request.

**Reply by scottrbaxter** [reply]
@bo0tzz For the time being, any chance there's an easy way to invert the current sorting method so that albums can be viewed by "oldest to newest" date? looking back through the project's history and I believe this used to be the default, but was changed to be more consistent elsewhere. I've tried to follow through where this is being set, but with refactoring over time it seems that the corresponding files have been heavily modified. A sorting solution (and the option to choose a default sorting method) would be great at some point, but right now I simply need photos in albums to be sorted ascending/sequentially by date. So far (I'm fairly new) Immich seems absolutely wonderful... thanks to all the devs who've put in work, it's really a great photo/video solution!

**Comment by Salindris** [comment]
Seeing as this is open since 2022, would it be possible to add a small workaround until this feature is implemented: just invert the current sort order, doesn't have to be configurable or anything. Albums will be viewed from oldest to newest photo 99.9% of the time so it would essentially solve most people's issue without having to code a configuration mechanism or anything.

**Reply by roserez** [reply]
I need this badly to. Just a reverse sort order. I'll hard code it for now if possible. I just need photos to display in forward chronological order. Reverse is worthless to me and confuses people. Does anyone have a patch?

**Reply by scottrbaxter** [reply]
I'm in the same situation. This software looks incredible and I can't wait to use it, but photos within albums having no way to be displayed in sequential order really is a deal breaker for all of my use cases. i'm hoping someone with a bit more context can recommend a solution, even as a temporary fix, until this feature can be fully implemented as intended.

**Reply by Soukyuu** [reply]
Please don't force sorting to oldest first. This breaks it for everyone who uses it as a photo backup solution, which may or may not be the majority. I'm as annoyed about not being able to set the sorting by default, but I'd be equally as annoyed for my main use case to break.

**Comment by lobotomm** [comment]
I am one more user who hopes that the album sort order gets reverted: oldest to newest seems to be right for 99,9%!

**Reply by Soukyuu** [reply]
Please don't force sorting to oldest first. This breaks it for everyone who uses it as a photo backup solution, which may or may not be the majority. I'm as annoyed about not being able to set the sorting by default, but I'd be equally as annoyed for my main use case to break.

**Comment by pakosalze** [comment]
+1

**Comment by Pazu5512** [comment]
Agreed, as a photo album tells a story, I think the default sorting option should be from oldest to newest. 

With the holy grail being this, ordering photos however we want
[Ordering.webm](https://github.com/immich-app/immich/assets/33207546/0c6937da-37b9-4f31-9082-19600c3328d6)

While it's mostly the case, a story is not always told from oldest to newest. Sometimes a photo you took a few minutes later is better appearing first. Maybe because the one you originally took had other people on it, or the light wasn't the best, and 10 minutes later, the conditions were better and thus you went back and took a new pic. For which a simple "older to newer" sort option wouldn't be sufficient to cover this case, and thus ruin the story you wanted to tell.

**Comment by aviv926** [comment]
> ### Feature detail
> It would be nice to have a sort option for the albums view. Google Photos currently provides three such options:
> 
> * Most recent photo - Album with the newest photo first
> * Last modified - Most recently modified album first
> * Album title - Alphabetical

Added to Immich.

> 
> It would also be nice to have similar sorting options inside of albums (and possibly on the main photos feed) - things like:
> 
> * Age (oldest/newest first)

Added with #7819.

> * Most recently added first
> * Type (photos or videos first)

**Reply by roserez** [reply]
Thank you!!!!

**Reply by aviv926** [reply]
> Thank you!!!!

The thanks are not to me, but to the development team ✌️

**Reply by lobotomm** [reply]
The new sorting album function seems only to work in the web view. Album order in app persists newest to oldest, no matter what I do in the web view.

**Reply by aviv926** [reply]
Yes, as stated in the new release 1.99 it currently supports the WEB version only.

**Reply by lobotomm** [reply]
I understand. I thought I can set the order in the web view and it stays that way in the app.

**Comment by chris114782** [comment]
I've just tried this for the first time, and the changes to ordering don't persist to shared links to the album.

If I set the order from oldest to newest, when I open the shared link to the album the web UI shows it in newest to oldest still.

**Comment by ok1ok1** [comment]
Please include a timeline view for albums on mobile. This will be especially beneficial if the upcoming automation features allow for "smart" albums.

**Comment by jstebenne** [comment]
Would this include ordering the pictures within an album using the original file names?

I am scanning a bunch of old photos and can easily order them with their file name while I am scanning. When I import those photos in Immich, the order currently seems random.

**Reply by bkiscool** [reply]
I would also like to see sorting by file names.

I have an album where photos are intended to be sorted by file name, but they are out of order because I can only change the display order by date created.

![image](https://github.com/user-attachments/assets/985abf2e-ae7f-4510-9644-e4aa4b20f15b)

**Reply by Soukyuu** [reply]
As @bkiscool says, I have a lot of albums in an external library which are basically trip reports. Sometimes I keep the jpg that comes out of the camera which then has a correct date taken, sometimes I have to process the raw file to get a better result. This however changes the "date taken", breaking the sorting. Sometimes also the EXIF data is missing completely (Yi 4k Camera).

What I do is rename the files with a numeric prefix to "fix" the display order instead of manually copying the date taken.

So the easiest sorting way for me would be "by filename". Though manual sorting by drag&drop would also be nice.

**Reply by piemanabc** [reply]
100% agree here, I primarily use Immich to view/store comics, and this is the last pain point for me. Importing these into albums is done automatically with something like [this ](https://github.com/Salvoxia/immich-folder-album-creator) which unfortunately means that albums are created in bulk, causing the images within to be added in a random order. 

an option for filename in a human readable format would be a game changer. When I say human readable I mean the order should be:

`1.jpg`, `3.jpg`, `4.jpg`, `10.jpg`, `13.jpg`. `21.jpg`, `22.jpg`

and not:

`10.jpg`, `13.jpg`. `1.jpg`, `21.jpg`, `22.jpg`, `3.jpg`, `4.jpg`

which seems to sort by the first character, and the length of the name? I've seen this with other image sorting/management software and it's effectively useless for my application.

**Comment by jomir1000** [comment]
a custom sorting method for albums would be really really nice... it could be as basic as an (editable) atribute "rank" for each pic. (sort by rank, then date)...

**Comment by Dezc** [comment]
In addition for the sort order on the album, that is already implemented, have the option in the parameter to define what is the default sort order on album creation.
Right now I have to go and edit the order on all album to be oldest first, a global configuration option should be an easy fix here :)

**Reply by marshalleq** [reply]
I just came here as I have wedding photos in an album and it starts the wrong way round. Most definitely I don't want the dance first scrolling down to the reception, exiting the church then kneeling at the altar lol. The trouble is this is for sharing and despite my being able to change my 'viewing' order it resets at next log in and it defaults the wrong way round for everyone. This feature needs to be persistent and be set in the album for all viewers. It would be also preferable to have a system global config that can set the default which can then be modified by users later. My default will be oldest first as I think that's what 99% will want but it will probably depend on if you're using as an iPhone photo backup or as an album sharing system mainly as I see those as two different use cases. 

Otherwise, nice product. Best open source image sharing I've found to date. Nice job everyone!

**Comment by WhyNotHugo** [comment]
Strong +1 here. Being able to sort photos chronologically would make a world of difference. Reverse chronological order doesn't make sense to me.

I don't even think this needs to be configurable—just sort photos chronologically by default. Does it ever make sense to have them in reverse chronological order?

**Reply by Soukyuu** [reply]
I am not sure if it's possible to make this the default for albums created while keeping the main view in reverse chronological view. If that's not possible, then I don't want the sorting to be changed, because that breaks my main use case, photo backup.

**Comment by marshalleq** [comment]
I think it makes sense if you're using a phone backup - i.e you want to see your latest photos at the top.  Almost a social feature.  But that's about it.

**Comment by WhyNotHugo** [comment]
On Tue, 10 Jun 2025, at 02:50, marshalleq wrote:
> 
> 
> I think it makes sense if you're using a phone backup - i.e you want to see your latest photos at the top. Almost a social feature. But that's about it.
> 

Why would you want to see them at the top? The photo gallery on the phone shows most recent photos at the bottom (given that it sorts photos in chronological order).

I’m not sure what “social feature” means here. Are you just trying to imitate what Facebook did?

**Reply by Soukyuu** [reply]
That is not what my gallery (Google Photos) is doing, and I likw how it is now

**Comment by WhyNotHugo** [comment]
On Tue, 10 Jun 2025, at 09:35, Ivan Pilipenko wrote:
> 
> 
> Please don't force sorting to oldest first. This breaks it for everyone who uses it as a photo backup solution, which may or may not be the majority. I'm as annoyed about not being able to set the sorting by default, but I'd be equally as annoyed for my main use case to break.
> 

Why would changing the order to be chronological order break backups?

**Reply by Soukyuu** [reply]
It would break the default view as it is now, not the backup itself.

**Comment by tanishqsindhu** [comment]
Able to do custom sorting method for albums would be really really nice like pin some important albums and then have date wise or other sorting pattern albums after pin albums

**Comment by SpeeterYT** [comment]
why are things like this not implemented for years?

**Comment by emaginsf** [comment]
+1 for manual sorting.
A primary goal of sharing albums includes "telling a story"
I went here, did this, then did that, look how nice the trip was....
Manual sort allows you to tell the story and share it.

**Comment by corvy** [comment]
When I sort by capture date it messes if I have had a burst shot (and I do that quite a bit). Sorting by file name would be a quick fix for me. Manual is also OK.

**Comment by wuyns** [comment]
+1
I have many ordered pics, sorting files by their name in album is important for me.

**Comment by cpresse** [comment]
I also would love to have sorting by file name for photos in albums. Manual sorting would also be okay.

**Comment by photoserver-pro** [comment]
+1 — strong support for this request.

Sorting is a fundamental baseline capability for any list/grid view and a standard expectation in modern photo management software. Without consistent sorting, navigation becomes unpredictable and inefficient, especially as libraries grow.

Sorting should be available and behave consistently across all relevant views (including ascending/descending), for example:
- Photos (timeline)
- Tags view
- People view
- Search results
- Similar photos
- Folder view

Common sort fields that cover the majority of use cases:
- Capture date (takenAt / CreateDate)
- File name
- Upload/Added date
- Rating (stars)
- Views/Visits
- Last modified / Last edited (updatedAt)

If feasible, persisting the selected sort per view (or globally) would also improve usability by avoiding repeated reconfiguration.


<img width="2279" height="1189" alt="Screenshot 2026-01-11 223232" src="https://github.com/user-attachments/assets/67e6d7e5-b925-4580-8aca-229f408e63b9" />

**Comment by photoserver-pro** [comment]
Quick update from my side: I’m still in “exploration mode” with Immich, and today I noticed that Albums already have sorting options.

Wouldn’t it make sense to offer the same sorting controls consistently across other views as well (People, Tags, Photos timeline, Places, Folders, etc.)? The functionality seems to exist already, so maybe it’s “just” about reusing it in more contexts.

Or am I missing a setting / hidden option that enables sorting elsewhere?

I also started a discussion on Discord to better understand real-world user use cases around sorting across views: https://discord.com/channels/979116623879368755/1460026887542931699/1460026887542931699

<img width="254" height="469" alt="image" src="https://github.com/user-attachments/assets/0428721c-e408-4f60-8494-5448b6a9a255" />
.
.
<img width="1944" height="428" alt="image" src="https://github.com/user-attachments/assets/a5b3b7dc-e42d-4360-98b0-0aa3c0e9627d" />

**Comment by scottrbaxter** [comment]
@photoserver-pro this actually has been a long-standing ask by many in the community... in fact, much longer than the album sorting feature itself has been around.

Some historical context: https://github.com/immich-app/immich/discussions/1689#discussioncomment-7960195 https://github.com/immich-app/immich/discussions/1689#discussioncomment-8119089

I agree with you that it seems like there could be some amount of repurposed code from the album sort for the assets within albums... though when the [album view sort order](https://github.com/immich-app/immich/pull/14648) was introduced as a separate functionality that did not use the album sort popup, i figured that was to provide a common request while they were still working out plans for the new mobile ui refactor.

I really hope they prioritize this feature soon. Sorting is a very standard feature for media solutions, and has been in my want list for so long. Sorting by size/length/type/count/dimensions/modified/recently accessed/etc. are all very important for storage considerations as well.

**Reply by photoserver-pro** [reply]
Thanks for the context @scottrbaxter! I’m fully aware that software development is always a prioritization trade-off with limited resources.

Based on ~8 years of experience with Piwigo and QNAP QuMagie, I find Immich’s approach—especially the AI features and the strong mobile integration—very inspiring. At the same time, I’m genuinely surprised by how some core “baseline” features that exist in almost every other photo solution are still missing.

Sorting is one of them. This is absolute standard functionality. And since sorting already exists in the Albums view, it’s hard for me to understand why it hasn’t been rolled out consistently across all relevant asset/result views (e.g., Tags, People, Places, Search results, Folders, Archive, etc.).

The second major gap (in a similar area) is the currently very rudimentary and inconsistent filter/faceted navigation. This is also standard in all solutions I know and becomes critical for efficiently exploring larger libraries.

More context here: https://github.com/immich-app/immich/discussions/24754#discussion-9268696

**Comment by marshalleq** [comment]
I've only been with immich for 1 year, but for a project that advertises as under heavy development it doesn't really seem like it.  Perhaps I'm missing something.  I keep thinking I should fork it and make something oriented to more of the photo market than whatever it is currently oriented toward - cause it seems to be completely lost on direction.  It's not just this feature.

**Reply by SpeeterYT** [reply]
I have the same impression, it just seems to have sk many little annoyances and missing features that just wont get fixed

**Reply by photoserver-pro** [reply]
Thanks @marshalleq for sharing your perspective. I’ve only been here for about four weeks and I’m still in the evaluation phase of Immich.

With ~8 years of experience using Piwigo, I’m impressed by many things Immich does extremely well (AI features, mobile integration, and overall performance). At the same time, I’m starting to feel like I may not fully understand Immich’s concept and target audience yet.

Seeing a feature request for a baseline capability like sorting remain open for years - especially when sorting already exists in the Albums view - does raise questions for me about the longer-term strategy and prioritization behind the product direction.

**Reply by marshalleq** [reply]
> Thanks @marshalleq for sharing your perspective. I’ve only been here for about four weeks and I’m still in the evaluation phase of Immich.

> With ~8 years of experience using Piwigo, I’m impressed by many things Immich does extremely well (AI features, mobile integration, and overall performance). At the same time, I’m starting to feel like I may not fully understand Immich’s concept and target audience yet.

I actually tried Piwigo before this and really wanted it to work. But the main issue was I couldn't get my family to use it due to how complex it looks to an average person. There was also some permissions challenges which I probably logged a ticket about which were a blocker for me. Immich has all the right beginnings.

**Comment by corvy** [comment]
If you have the time to fork and run with it, why not make a PR and suggest the improvement here? Separating into a fork makes no sense to me.

**Comment by marshalleq** [comment]
My thinking is that it's a directional challenge.  I get the feeling that whoever is leading this was more interested in make a storage location for their photos to replace google photos - which is a fair thing to want to do.  But I don't think there's really an appetite to take it to a bit more of a photo management application at present.  I've sort of been biding my time a bit at the moment due to other things on, but the time may well come.  But if the team is keen on taking it into a direction where stories can be told through photos and some basic management things like even just a rotate photo option OMG, I couldn't believe it when I saw that was missing.

**Comment by SpeeterYT** [comment]
I really feel like we need a month of wth or similar

**Comment by marshalleq** [comment]
WTH?

**Reply by SpeeterYT** [reply]
https://www.home-assistant.io/blog/2024/11/30/the-month-of-what-the-heck/

**Comment by corvy** [comment]
I agree to this last part, but again I am more leaning to voicing and contribute rather than forking off.

**Comment by marshalleq** [comment]
I agree, forking is the worst outcome, but if there are deaf ears no amount of yelling will change anything.  I'm not ready, so they have that time I guess.  And I also don't know what the sentiment is around here regarding it, certainly there are a few or us that have been requesting things things for some time now - this was logged in July 2022!  Anyone know if there's a roadmap or anything?  it'd be good to know if there is any actual vision that is supposedly being worked to and everything is just stalled, or if there's no vision and agreement to go anyway, the former be bad but understandable for a free commitment, the latter being worse and probably explains more of what we're seeing.

**Comment by photoserver-pro** [comment]
I think this is the core question here: what is the target audience and intended purpose of the Immich project?

When I discovered Immich about four weeks ago, the website messaging was very clear:
	•	organize your photos
	•	manage your photos
	•	browse your photos
	•	search your photos

I’ve been using Lightroom since 2005, ran Piwigo for the last ~8 years, and also have extensive hands-on experience with QNAP QuMagie. In that context, capabilities like sorting and faceted/filter navigation are absolute baseline requirements (more context on filtering here: https://github.com/immich-app/immich/discussions/24754#discussion-9268696).

Right now I’m genuinely torn on whether Immich is the right long-term solution for my 600k+ family photo archive. Having large, flat lists of photos without being able to reliably filter and sort them is poor UX—and in practice becomes hard to use.

I’m not a developer, but I do bring ~20 years of professional e-commerce and search experience, and in that world sorting and filtering are foundational UX building blocks.

I’d really appreciate a better understanding of where Immich is heading in terms of product direction and priorities, so I can assess whether it makes sense to invest more time and resources (I’ve already built a dedicated hardware setup for testing) into adopting Immich.


![image](https://github.com/user-attachments/assets/94fe6309-ca30-42dd-a3bd-98434ee4f230)

**Comment by vectorjohn** [comment]
I see that albums already support sorting by "newest first" or "oldest first". Whatever field that is based on (file creation, date added, modified, etc), it's just toggling the direction.
I think what would solve all these conundrums is to change that from just a direction to a "mode" and a direction. I.e.
sort by: "date added" / "date created" / "date modified"
direction: "ascending" / "descending" (or oldest/newest first if people prefer that terminology.

I think this change would easily set things up to support manual sorting, which should be a different feature request because it would require changing other tables and a whole lot more UI work. But what that would look like is one more sort mode, "manual". And then each of the asset_album database links would also contain a sort order for each photo. That would get updated by manual drag and drop.
The complicated part comes when toggling between date based and manual sorting, what do we do with the existing order, etc. That's another reason it's more complicated.

But just adding more sort modes (sorting by different fields) should be a starting point. I'm willing to work on it. I've looked at the database code and some services and it looks reasonably easy, but I haven't dug really deep in there yet to know.

**Comment by teconomix** [comment]
I want to second the feature request towards manual sorted albums. That is of utmost importance to me. I often take pictures in workshops and need to sort them afterwards to represent the results of the workshops. 

It would be great to also have text in between to mark separate parts of the workshop, just like in google fotos.

**Reply by chris400** [reply]
There is a dedicated feature request for manual re-ordering. You might want to place your vote there
https://github.com/immich-app/immich/discussions/2208

**Comment by millercentral** [comment]
Our use case for custom ordering for albums is scanned photos. We have recently scanned/archived a large amount of old family photos and these scans don't have real metadata in terms of date taken etc. We want to share these photos to family members, manually ordering the pictures in an album around specific people/events/etc, but messing with the metadata of each photo individually is not a viable solution. This would be a very impactful feature to us.

**Reply by chris400** [reply]
There is a dedicated feature request for manual re-ordering. You might want to place your vote there
https://github.com/immich-app/immich/discussions/2208

**Comment by mariolino320** [comment]
I’d like to add my support to this request; I need to be able to sort photos within an album alphanumerically, rather than just by date. Thanks.

**Comment by babakfp** [comment]
Why do we need to beg for the most basic features!

**Reply by babakfp** [reply]
`/recently-added` this page doesn't even do what its title says it's sposed to do!

**Reply by Soukyuu** [reply]
Because you're not paying them to implement it. You want it? Submit a PR. Nobody needs entitled whining.

**Comment by Merlin-2504** [comment]
Alphabetical sorting for the People view would be very useful.

With a large photo library and many named people, finding a specific person becomes increasingly difficult.

It would be helpful to have a sorting option in People, for example:

* Name A–Z
* Name Z–A
* Number of photos
* Named / unnamed people

Ideally, Immich should remember the selected sorting method.

**Comment by babakfp** [comment]
lol, they removed user reactions from my comments! didn't know we are dealing with this kind of people! we didn't like what you said, no criticisms! can't even talk in a damn place called discussions! this tells me to not waste my time around here.

**Reply by pakosalze** [reply]
I've had similar observations from this forum. The moderator here isn't listening. He knows better... And these messages will be deleted because they're "off-topic" ;)

**Comment by Merlin-2504** [comment]
Unfortunately, posts and comments seem to get deleted quite often. That’s a shame, because it makes a real discussion difficult.
Of course, we should always keep in mind that many people involved with Immich contribute their time and effort voluntarily. That absolutely deserves respect and appreciation.
At the same time, feedback, criticism, and suggestions from real-world users are important. Our experiences and observations can also help improve Immich and move the project forward.

**Reply by bo0tzz** [reply]
Nothing is being deleted. Some comments that are low quality/off-topic/redundant get hidden (but can still be viewed just fine) so the thread doesn't get cluttered.

**Comment by brueske** [comment]
Shocked that sorting by reverse chronological order doesn't exist yet. This seems embarrassing that such a elementary feature hasn't been added despite years of requests

**Reply by hunnguye** [reply]
Don't know about your deployment but for me that exists both on Android and web.

I agree that more sorting options would be nice, but chronological sorting already exists.

Web> options > display order (drop-down)
Android > options > change display order
